trunner/runners/IMXRT106xRunner.py

```diff
@@ -89,72 +72,5 @@ def reboot(self, serial_downloader=False, cut_power=False):
         else:
             self._restart_by_jtag()
 
-    def flash(self):
-        self.reboot(serial_downloader=True, cut_power=self.is_cut_power_used)
-
-        phd = None
-        try:
-            with PloTalker(self.port) as plo:
-                Psu(script=self.SDP).run()
-                plo.wait_prompt()
-                with Phoenixd(self.phoenixd_port) as phd:
-                    plo.copy_file2mem(
-                        src='usb0',
-                        file=self.IMAGE,
-                        dst='flash1',
-                        off=0
-                    )
-        except (TIMEOUT, EOF, PloError, PhoenixdError) as exc:
-            exception = f'{exc}\n'
-            if phd:
-                exception = phd_error_msg(exception, phd.output())
-
-            logging.info(exception)
-            sys.exit(1)
-
-        self.reboot(cut_power=self.is_cut_power_used)
-
-    def load(self, test):
-        """Loads test ELF into syspage using plo"""
-
-        phd = None
-        load_dir = str(rootfs(test.target) / 'bin')
-        self.reboot(cut_power=self.is_cut_power_used)
-        try:
-            with PloTalker(self.port) as plo:
-                # Because of powering all Rpi ports after powering the board,
-                # there is need to second reboot (without cut power) in order to capture all data
-                # (when using _restart_by_poweroff)
-                if self.is_cut_power_used:
-                    self.reboot(cut_power=False)
-                plo.wait_prompt()
-
-                if not test.exec_cmd:
-                    # We got plo prompt, we are ready for sending the "go!" command.
-                    return True
-
-                with Phoenixd(self.phoenixd_port, dir=load_dir) as phd:
-                    plo.app('usb0', test.exec_cmd[0], 'ocram2', 'ocram2')
-        except (TIMEOUT, EOF, PloError, PhoenixdError) as exc:
-            if isinstance(exc, PloError) or isinstance(exc, PhoenixdError):
-                test.exception = str(exc)
-                test.fail()
-            else:  # TIMEOUT or EOF
-                test.exception = Color.colorify('EXCEPTION PLO\n', Color.BOLD)
-                test.handle_pyexpect_error(plo.plo, exc)
-
-            if phd:
-                test.exception = phd_error_msg(test.exception, phd.output())
-
-            return False
-
-        return True
-
     def run(self, test):
-        if test.skipped():
-            return
-
-        if not self.load(test):
-            return
-
         super().run(test)
```

Comment:
If all that `run()` does is running `super().run()` then it is not unnecessary redefinition. You can omit it

Reply:
That's right, thanks